scout/server/blueprints/variant/utils.py

```diff
@@ -2,7 +2,15 @@
 from typing import Dict, List, Optional, Tuple
 
 from scout.adapter import MongoAdapter
-from scout.constants import ACMG_COMPLETE_MAP, CALLERS, CLINSIG_MAP, SO_TERMS, VARIANT_FILTERS
+
```

Comment:
We also run `black` with a line length of 100. But we have a write enabled linter action to fix that; we haven't fixed the `isort` since it went unsupported by an update to that linter action. Maybe we will switch to `ruff` in a bit https://docs.astral.sh/ruff/formatter/#sorting-imports. 😁